roles/openshift_facts/library/openshift_facts.py

```diff
@@ -56,6 +56,13 @@ def migrate_docker_facts(facts):
     if 'node' in facts and 'portal_net' in facts['node']:
         facts['docker']['hosted_registry_insecure'] = True
         facts['docker']['hosted_registry_network'] = facts['node'].pop('portal_net')
+
+    # log_options was originally meant to be a comma separated string, but
+    # we now prefer an actual list, with backward compatability:
+    if 'log_options' in facts['docker'] and \
+            isinstance(facts['docker']['log_options'], basestring):
+        facts['docker']['log_options'] = facts['docker']['log_options'].split(",")
```

Comment:
@dgoodwin The openshift_docker_facts role is applied prior to the docker role, so in the case of a comma separated list:
- We would persist the comma separated list the first time openshift_docker is run
- We would pass the persisted comma separated list to the docker role as docker_log_options and convert the value using the oo_split filter
- We would convert the value from a comma separated list to an array the next time openshift_facts is run on the host